src/main/scala/Game.scala

```diff
@@ -52,10 +56,9 @@ case class Game(
     )
   }
 
-  private def applyClock(lag: Centis, withInc: Boolean) = clock map {
-    case c: RunningClock => c.step(lag, withInc)
-    case c: PausedClock if (turns - startedAtTurn) == 1 => c.start.switch
-    case c => c.switch
+  private def applyClock(lag: MoveMetrics, withInc: Boolean) = clock.map {
+    case c if c.isRunning => c.step(lag, withInc)
+    case c => if (turns - startedAtTurn == 1) c.switch.start else c.switch
```

Comment:
would it be clearer as:
```scala
  case c if (turns - startedAtTurn == 1) => c.switch.start
  case c => c.switch
```
I wasn't sure what this code was for, it seemed hacky